src/Akeneo/Category/back/Application/Converter/Checker/ValueCollectionRequirementChecker.php

```diff
@@ -164,6 +164,6 @@ private static function assertImageData(?array $imageData): void
      */
     private static function assertTextData(string $textData): void
     {
-        Assert::stringNotEmpty($textData, "Expected String and not empty value for 'data'");
+        Assert::nullOrString($textData, "Expected String or null value for 'data'");
```

Comment:
```suggestion
        Assert::string($textData, "Expected String or null value for 'data'");
```
The null value is reserved only when we have array and it is manage here : https://github.com/akeneo/pim-community-dev/blob/451cd9600d5497a009dc3116f6ecdc01692cb017/src/Akeneo/Category/back/Application/Converter/Checker/ValueCollectionRequirementChecker.php#L129. Functionaly, we cannot have a text set to null. This behavior will be manage when we will implemente the Value object in ValueCollection.

Reply:
True @antoinetrouve , thanks and well done, I make the changes